tests/Set/Symfony60/Fixture/route_loader_already_using_mixed.php.inc

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
please rename fixture to something like: `skip_loader_already_using_mixed.php.inc`